utils/vlog/vlog_to_model.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
+"""
+Convert a Verilog simulation model to a VPR `model.xml`
+
+The following Verilog attributes are considered on ports:
+    - `(* CLOCK *)` : force a given port to be a clock
```

Comment:
At the moment I look at connections to the clock inputs of $dff cells, which is not ideal, for auto-detection. Just using a pin named `clk` doesn't work because of BRAM cells or IO cells (in the iCE40) with multiple clocks.